x-pack/plugins/fleet/server/services/epm/elasticsearch/template/template.ts

```diff
@@ -542,7 +542,9 @@ function getBaseTemplate({
         _meta,
       },
     },
-    data_stream: { hidden },
```

Comment:
I wonder how did this even work? Wouldn't this render as `data_stream: { true/false }` instead of `data_stream: { hidden: true/false }`?

Reply:
This is [javascript shorthand](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#syntax) when an object property and local variable have the same name that essentially expands to 

```js
data_stream: { 
  hidden: hidden
}
```